ui/components/app/add-network/add-network.js

```diff
@@ -63,29 +69,70 @@ const AddNetwork = ({
           color={COLORS.UI3}
           margin={[4, 0, 3, 0]}
         >
-          {t('customNetworks')}
+          {t('popularCustomNetworks')}
         </Typography>
         {nets.map((item, index) => (
           <Box
             key={index}
             display={DISPLAY.FLEX}
             alignItems={ALIGN_ITEMS.CENTER}
+            justifyContent={JUSTIFY_CONTENT.SPACE_BETWEEN}
             marginBottom={6}
           >
-            <img
-              className="add-network__token-image"
-              src={item?.rpcPrefs?.imageUrl}
-              alt={t('logo', [item.ticker])}
-            />
-            <Typography variant={TYPOGRAPHY.H7} color={COLORS.BLACK}>
-              {item.ticker}
-            </Typography>
-            <img
-              className="add-network__add-icon"
-              src="./images/times.svg"
-              alt={`${t('add')} ${item.ticker}`}
-              onClick={onAddNetworkClick}
-            />
+            <Box display={DISPLAY.FLEX} alignItems={ALIGN_ITEMS.CENTER}>
+              <IconBorder size={24}>
+                <IconWithFallback
+                  icon={item.rpcPrefs.imageUrl}
+                  name={item.nickname}
+                  size={24}
+                />
+              </IconBorder>
+              <Typography
+                variant={TYPOGRAPHY.H7}
+                color={COLORS.BLACK}
+                fontWeight={FONT_WEIGHT.BOLD}
+                boxProps={{ marginLeft: 2 }}
+              >
+                {item.nickname}
+              </Typography>
+            </Box>
+            <Box display={DISPLAY.FLEX} alignItems={ALIGN_ITEMS.CENTER}>
+              {!regexp.test(item.rpcUrl) && (
```

Comment:
Could you leave a comment explaining what's happening here?

Reply:
This will filter all networks that doesn't use infura.io as the rpcUrl and it will show a warning next to that network. 
It is done to fulfil acceptance criteria
Acceptance criteria point from the ticket: `Include new tool tip icon for all networks which RPC url doesn't belong to Infura `

Acceptance criteria point from the Notion: `There’s a clear differentiation on the network list between networks that are supported by Infura and the ones that are not`